pkg/chunk/grpc/storage_client.go

```diff
@@ -0,0 +1,124 @@
+package grpc
+
+import (
+	"context"
+	"io"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+
+	"github.com/cortexproject/cortex/pkg/chunk"
+)
+
+type StorageClient struct {
+	cfg        Config
+	schemaCfg  chunk.SchemaConfig
+	client     GrpcStoreClient
+	connection *grpc.ClientConn
```

Comment:
Not used. Can be removed.

Reply:
This comment is still valid. `connection` is unused, but I think we should use it to close the connection in `Stop()`.